lusid-sdk-python/tests/test_transparency_strategies.py

```diff
@@ -558,7 +558,11 @@ def aggregate_portfolio_group(self):
                                                                                  op='sum'),
                                                             models.AggregateSpec(key='Holding/default/Cost',
                                                                                  op='sum')
-                                                            ])
+                                                            ],
```

Comment:
I think the intent of the test was to return all the data - so should we change this to op="Value" and remove the groupby - might be a question for @MikeMcGarry

Reply:
The intended test is to return a row for each instrument, which we need to groupby to achieve.